src/Resources/config/services/order_processing.xml

```diff
@@ -8,5 +8,21 @@
             <argument type="service" id="setono_sylius_order_edit.order_processing.order_payment_processor.inner"/>
             <argument type="service" id="request_stack"/>
         </service>
+
+        <service id="Setono\SyliusOrderEditPlugin\Provider\OldOrderProvider">
```

Comment:
If we change service ids to fqcns it should be consistent. However, I don't know why I haven't switched to that yet. Probably because there are still so many services with those snake cased service names

Reply:
I got used to using FQCN already for a long time. Anyway, it's up to you, it's your library 😄 It's only a matter of convention, doesn't change anything 🚀

Reply:
For now it's easier to keep the old way, because all other services use that :D

But this is something I will change in the future I think :D 

👍